src/org/zaproxy/zap/extension/api/CoreAPI.java

```diff
@@ -838,9 +843,21 @@ public void process(Alert alert) {
 			processAlerts(
 					this.getParam(params, PARAM_BASE_URL, (String) null), 
 					this.getParam(params, PARAM_START, -1), 
-					this.getParam(params, PARAM_COUNT, -1), counter);
+					this.getParam(params, PARAM_COUNT, -1), 
+					this.getParam(params, PARAM_RISK, -1), counter);
 			
 			result = new ApiResponseElement(name, Integer.toString(counter.getCount()));
+		} else if (VIEW_ALERTS_SUMMARY.equals(name)) {
+			final ApiResponseList resultList = new ApiResponseList(name);
+			Map<String, Object> alertData = new HashMap<>();
+			int[] riskIdArray = {Alert.RISK_HIGH, Alert.RISK_MEDIUM, Alert.RISK_LOW, Alert.RISK_INFO};
+			CounterProcessor<Alert> counter = new CounterProcessor<>();
+			for (int riskId:riskIdArray) {
+				processAlerts(this.getParam(params, PARAM_BASE_URL, (String) null), -1, -1, riskId, counter);
+				alertData.put(Alert.MSG_RISK[riskId], String.valueOf(counter.getCount()));
+			}
+			resultList.addItem(new ApiResponseSet<Object>("risk", alertData));
```

Comment:
Same as is now but without the list:
```JSON
{"alertsSummary":{"High":0,"Low":4,"Medium":3,"Informational":0}}
```

That can be done, for example, with:
```Java
new ApiResponseSet<Integer>(name, alertData) {
    @Override
    public net.sf.json.JSON toJSON() {
        JSONObject jo = new JSONObject();
        jo.put(getName(), super.toJSON());
        return jo;
    };
};
```
(having this assign to the `result`.)